debug.xml

```diff
@@ -320,6 +323,10 @@
         DELETE FROM notifications WHERE id = :id;
     </entry>
 
+    <entry key='database.clearPositionsHistory'>
+        DELETE FROM positions WHERE deviceid = :deviceId and servertime &lt; :serverTime;
```

Comment:
it never delete the last position.
it count days back from the last update date and not from current date:

Date dateBefore = new Date(lastUpdate.getTime() - histDays * 24 * 3600 * 1000 );